e3/vcs/svn.py

```diff
@@ -46,6 +46,54 @@ def __init__(self, working_copy):
         """
         self.working_copy = working_copy
 
+    @classmethod
+    def local_url(self, repo_path):
+        """Return the url of a svn repository hosted locally.
+
+        :param repo_path: path to the repo
+        :type repo_path: str
+        :return: the url that can be used as repository url
+        :rtype: str
+        """
+        repo_path = os.path.abspath(repo_path)
+        if sys.platform == 'win32':  # unix: no cover
+            if len(repo_path) > 1 and repo_path[1] == ':':
+                # svn info returns the URL with an uppercase letter drive
+                repo_path = repo_path[0].upper() + repo_path[1:]
+            return 'file:///' + repo_path.replace('\\', '/')
+        else:  # windows: no cover
+            return 'file://' + repo_path
+
+    @classmethod
+    def create(cls, repo_path, initial_content_path=None):
```

Comment:
Food for thought. I just realized while reading the testcases that, with SVN, the "create" method creates a different kind of SVN repository -- in other words, the word "repository" is overloaded; it could me the central copy that users check out from and push to (what this method creates), vs the "working copy" (what this class works with).

I think it's worth enhancing the documentation to be clearer about this.